coll/jq.go

```diff
@@ -0,0 +1,35 @@
+package coll
+
+import (
+	"fmt"
+
+	"github.com/itchyny/gojq"
+)
+
+// JQ -
+func JQ(jqExpr string, in interface{}) (interface{}, error) {
+	query, err := gojq.Parse(jqExpr)
+	if err != nil {
+		return nil, fmt.Errorf("couldn't parse JQ %s: %w", jqExpr, err)
+	}
+	iter := query.Run(in)
```

Comment:
Just checking the docs for gojq it looks like there's a `query.RunWithContext` - I'd rather use this so we can support cancellation wherever possible.

I've made this change locally and will push a commit to your branch.